README.md

```diff
@@ -19,6 +19,11 @@ git submodule init
 git submodule update
 ```
 
+You must run this command once to enable modules.
+```
+./ccd enable backend frontend  sidam sidam-local sidam-local-ccd dm-store
```

Comment:
You have two spaces but not a big deal if you do not want to clean that up.

Reply:
i will fix indeed :)